.github/workflows-source/ci-main.yml

```diff
@@ -235,6 +253,39 @@ jobs:
           # Clean up the output base for the next run
           sudo rm -rf /var/tmp/bazel-output
 
+  pocketic-on-windows:
+    name: PocketIC tests on Windows
+    needs: [bazel-test-all]
+    <<: *pocketic-windows-trigger
+    <<: *windows-setup
+    steps:
+      - <<: *checkout
+      - name: Download PocketIC server
+        uses: actions/download-artifact@v4
+        with:
+          name: pocket-ic-server
+      - name: Setup WSLv2
+        uses: vedantmgoyal9/setup-wsl2@main
+      - name: Code linting
+        run: |
+          cargo clippy --locked --all-features -p pocket-ic -p pocket-ic-test-canister --all-targets -- -D warnings -D clippy::all -D clippy::mem_forget -C debug-assertions=off
```

Comment:
Isn't this redundant since we already have a cargo clippy job?

Reply:
We should also run cargo clippy on windows due to conditional compilation `#[cfg(windows)]` used at some places in PocketIC (library) code.